lib/configuration/defaults.ts

```diff
@@ -5,6 +5,8 @@ export const defaultConfiguration: Required<Configuration> = {
   sourceRoot: 'src',
   collection: '@nestjs/schematics',
   entryFile: 'main',
+  projects: {},
+  monorepo: false,
```

Comment:
I'm not entirely sure if setting `projects` to `{}` is safe (previously, this value was `undefined` so there might be a check somewhere that compares this value explicitly to `undefined`)

Reply:
🐈